tests/test_views.py

```diff
@@ -3631,16 +3631,21 @@ def test_external_login_email_get_with_invalid_session(self):
 
     def test_external_login_email_get_with_another_user_logged_in(self):
         another_user = AuthUserFactory()
-        url = self.user.get_confirmation_url(self.user.username, external_id_provider='service')
+        url = self.user.get_confirmation_url(self.user.username, external_id_provider='service', destination='dashboard')
```

Comment:
update relevant tests that User.get_confirmation_url() takes `destination` parameter